app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,27 @@
+<%# <%- if controller_name != 'sessions' %> 
+  <%# <%= link_to "Log in", new_session_path(resource_name) %><br />
+<%# <% end %> 
+
+<%# <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%# <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+<%# <% end %>
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
@BenMukebo You are absolutely right, thank you very much for your comments. I have already applied your comments and applied your advice.

Reply:
This change has been done well